examples/pytorch/Logging/pytorch log model.ipynb

```diff
@@ -0,0 +1,416 @@
+{
```

Comment:
let's use lower case for file path: `examples/pytorch/logging/`

and put underscore in the file name: `pytorch_log_model.ipynb`